src/flat-button.jsx

```diff
@@ -26,6 +26,7 @@ let FlatButton = React.createClass({
     disabled: React.PropTypes.bool,
     hoverColor: React.PropTypes.string,
     label: validateLabel,
+    labelPosition: React.PropTypes.oneOf(['left', 'right']),
```

Comment:
I think these should be one of `before` or `after`. Since that is a truer indication of what it's doing.